src/test/java/com/github/onsdigital/babbage/api/filter/ShortUrlFilterTest.java

```diff
@@ -116,26 +125,26 @@ public void shouldNotLoadShortcutMappingAndShouldRedirect() throws Exception {
 		boolean result = filter.filter(request, response);
 
 		assertThat("Expected filter to return false but was true. Test failed.", result, equalTo(false));
-		verifyZeroInteractions(shortcutUrlService);
-		verify(response, times(1)).sendRedirect(SHORT_URL_REDIRECT);
-		verify(response, times(1)).setStatus(HttpServletResponse.SC_MOVED_PERMANENTLY);
+		verifyNoInteractions(shortcutUrlService);
+		verify(response, times(1)).setHeader(HttpHeaders.LOCATION, SHORT_URL_REDIRECT);
+		verify(response, times(1)).setStatus(HttpStatus.SC_PERMANENT_REDIRECT);
 	}
 
-	@Test(expected = IOException.class)
-	public void shouldThrowIOExceptionWhenSettingRedirect() throws Exception {
+	@Test
+	public void shouldHandleErrorIfExceptionThrown() throws Exception {
 		when(request.getRequestURI())
 				.thenReturn(SHORT_URL);
 
 		doThrow(new IOException())
-				.when(response).sendRedirect(anyString());
+				.when(shortcutUrlService).shortcuts();
 
-		TestsUtil.setPrivateStaticField(filter, "shortcuts", mapOptional);
+		try (MockedStatic<ErrorHandler> errorHandlerMock = mockStatic(ErrorHandler.class)) {
+			boolean result = filter.filter(request, response);
+
+			assertEquals(true, result);
```

Comment:
This can be simplified:
```suggestion
			assertTrue(result);
```

Reply:
Looks like you've not got `$DP_REPO_DIR` set maybe?